x/auth/client/rest/query.go

```diff
@@ -109,6 +135,17 @@ func QueryTxsRequestHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 
 // QueryTxRequestHandlerFn implements a REST handler that queries a transaction
 // by hash in a committed block.
+//
+// @Summary Query tx by hash
+// @Description Retrieve a transaction using its hash.
+// @Tags Transactions
+// @Produce json
+// @Param hash path string true "tx hash"
+// @Param height query string false "Block height to execute query (defaults to chain tip)"
+// @Success 200 {object} rest.txSearch
```

Comment:
why is the `rest.` necessary if you're on the same pkg?

Reply:
yes, it's necessary. this is how swag finds the types.